client/app/components/pages/Home.vue

```diff
@@ -19,8 +21,17 @@
           v-bind:selectedTranscript="selectedTranscript"
           v-on:transcript-selected="onTranscriptSelected"
           v-on:gene-source-selected="onGeneSourceSelected"
-          v-on:gene-region-buffer-change="onGeneRegionBufferChange">
+          v-on:gene-region-buffer-change="onGeneRegionBufferChange"
+          v-on:gene-region-zoom="onGeneRegionZoom"
+          v-on:gene-region-zoom-reset="onGeneRegionZoomReset"
```

Comment:
There are vue shorthands for v-bind and v-on (e.g. `:selectedTranscript` and `@gene-region-buffer-change`.